pkg/products/monitoring/reconciler.go

```diff
@@ -644,13 +644,10 @@ func (r *Reconciler) reconcileAlertManagerConfigSecret(ctx context.Context, serv
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("serviceKey is undefined in pager duty secret")
 	}
 
-	// handle dead mans snitch credentials
-	dmsSecret := &corev1.Secret{}
-	if err := serverClient.Get(ctx, types.NamespacedName{Name: r.installation.Spec.DeadMansSnitchSecret, Namespace: rhmiOperatorNs}, dmsSecret); err != nil {
-		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("could not obtain dead mans snitch credentials secret: %w", err)
-	}
-	if len(dmsSecret.Data["url"]) == 0 {
-		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("url is undefined in dead mans switch secret")
+	//Get dms  credentials
```

Comment:
Tiny little detail -> 2 white spaces between `dms` and `credentials` in the comment.

Reply:
Agreed. Fixed.